README.md

```diff
@@ -18,6 +18,8 @@ might help with the notation for mathematical expressions.
 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
 
-Therefore there is some constant k in c that satisfies $\forall n\ge n_0, f(n) < k g(n)$
+So choosing some arbituary constant in c, k satisfies $\forall n\ge n_0, f(n) < k g(n)$
```

Comment:
Don't know what you mean here. $c$ is not a set, so how can we choose something from it?